genie/genie_elt.py

```diff
@@ -0,0 +1,80 @@
+"""GENIE ELT pipeline"""
+import os
+
+from dotenv import dotenv_values
+import pandas as pd
+import snowflake.connector
+from snowflake.connector.pandas_tools import write_pandas
+import synapseclient
+
+
+def main():
+    """GENIE ELT pipeline"""
+    syn = synapseclient.login()
+
+    config = dotenv_values("../.env")
+
+    ctx = snowflake.connector.connect(
+        user=config['user'],
+        password=config['password'],
+        account=config['snowflake_account'],
+        database="genie",
+        role="SYSADMIN",
+        warehouse="compute_xsmall"
+    )
+
+    cs = ctx.cursor()
+    # data_CNA
+    structured_data = (
+        "data_clinical", "data_mutations",
+        "assay_information", "data_cna_hg19", "data_gene_matrix",
+        "data_sv", "genomic_information"
+    )
+    releases = syn.getChildren("syn7844529")
+    for release in releases:
+        if release['name'] != "Release 15.0-public":
+            continue
+        print(release['name'])
+        release_name = (release['name']
+            .replace("Release ", "")
+            .replace(".", "_")
+            .replace("-public", "")
+        )
+        release_id = release['id']
+        release_files = syn.getChildren(release_id)
```

Comment:
By default this may include more than just the Files you intend to retrieve:
https://python-docs.synapse.org/reference/client/?h=getchildren#synapseclient.Synapse.getChildren